Libraries/LibCore/ArgsParser.cpp

```diff
@@ -324,6 +324,29 @@ void ArgsParser::add_positional_argument(int& value, const char* help_string, co
     add_positional_argument(move(arg));
 }
 
+// Theoritically this is bad. In practice - this is not that harmful :)
+static constexpr bool isnan(double __x) { return __builtin_isnan(__x); }
```

Comment:
I don't understand this comment. Why is this theoretically bad?
We use `__builtin_isnan()` elsewhere already.

Reply:
Sorry for late reply.

I only see it here: https://github.com/SerenityOS/serenity/blob/6075defd55e66a3c627cb6912a1431971eaa1ed6/Libraries/LibJS/Runtime/Value.h#L78 (and in the toolchain itself).

I am worried by the fact that `#include <math.h>`  (or `cmath`) fails. I am also worried that this will lock serenityOS to gcc, and moving to clang or a self made compiler (wtf not, this project already did everything else).